apps/analytics/activityreports.py

```diff
@@ -0,0 +1,58 @@
+from superdesk.resource import Resource
+from superdesk import get_resource_service
+from eve.utils import ParsedRequest
+import json
+from superdesk.services import BaseService
+
+
+class ActivityReportResource(Resource):
+    """Activity Report schema
+    """
+
+    schema = {
+        'desk': Resource.rel('desks', nullable=True),
+        'operation': {
+            'type': 'string'
+        },
+        'date': {
+            'type': 'datetime'
+        },
+        'report': {
+            'type': 'dict'
+        }
+    }
+    item_methods = ['GET', 'DELETE']
+    resource_methods = ['POST']
+    privileges = {'POST': 'activityreports', 'DELETE': 'activityreports', 'GET': 'activityreports'}
+
+
+class ActivityReportService(BaseService):
+
+    # TO DO: add the date as the third search parameter
+    def search_items(self, desk, operation):
+        query = {
+            "query": {
+                "filtered": {
+                    "filter": {
+                        "bool": {
+                            "must": [
+                                {"term": {"operation": operation}},
+                                {"term": {"task.desk": str(desk)}},
+                            ]
+                        }
+                    }
+                }
+            }
+        }
+        request = ParsedRequest
+        request.args = {'source': json.dumps(query), 'repo': 'archive,published,archived,ingest'}
+        items_list = list(get_resource_service('search').get(req=request, lookup=None))
```

Comment:
Does anyone know what I've been missing here? When using the `search` endpoint I'm getting 0 items in the report but when I explicitly use `archive` endpoint, the report is being populated with the correct number of items.